proc_blocks/audio_float_conversion/src/lib.rs

```diff
@@ -0,0 +1,95 @@
+#![no_std]
+
+extern crate alloc;
+
+#[cfg(test)]
+#[macro_use]
+extern crate std;
+
+use alloc::vec::Vec;
+use runic_types::{HasOutputs, Tensor};
+use rune_pb_core::{ProcBlock, Transform};
+
+// TODO: Add Generics
+
+#[derive(Debug, Clone, PartialEq, ProcBlock)]
+pub struct AudioFloatConversion {
```

Comment:
We should probably also tell the system what transformations are supported. 

```suggestion
#[transform(input = [i16; _], output = [f32; _])]
pub struct AudioFloatConversion {
```

That way a `rune inspect ./my_proc_block` command would be able to inspect the proc block and tell the user what it can do without needing to read the Rust source code.